release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -131,6 +131,46 @@ For more information, see xref:../networking/openshift_network_security/configur
 [id="ocp-4-16-storage"]
 === Storage
 
+[id="ocp-4-16-storage-clone-azure-file"]
+==== Volume cloning supported for Azure File (Technology Preview)
+{product-title} 4.16 introduces volume cloning for the Microsoft Azure File Container Storage Interface (CSI) Driver Operator as a Technology Preview feature. Volume cloning duplicates an existing persistent volume (PV) to help protect against data loss in {product-title}.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file[Azure File CSI Driver Operator] and xref:..//storage/container_storage_interface/persistent-storage-csi-cloning.adoc#persistent-storage-csi-cloning[CSI volume cloning].
+
+[id="ocp-4-16-storage-node-expansion-secret"]
+==== Node Expansion Secret is generally available
+The Node Expansion Secret feature allows your cluster to expand storage volumes, even when access to those volumes requires a secret (for example, a credential for accessing a Storage Area Network (SAN) fabric) to perform the node expand operation. {product-title} 4.16 supports this feature as generally available.
+
+[id="ocp-4-16-vsphere-max-snapshot"]
+==== Changing vSphere CSI maximum number of snapshots is generally available
+The default maximum number of snapshots in VMWare vSphere Container Storage Interface (CSI) is 3 per volume. In {product-title} 4.16, you can now change this maximum number of snapshots to a maximum of 32 per volume. You also have granular control of the maximum number of snapshots for vSAN and Virtual Volume datastores. {product-title} 4.16 supports this feature as generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#vsphere-change-max-snapshot_persistent-storage-csi-vsphere[Changing the maximum number of snapshots for vSphere]
+
+[id="ocp-4-16-pv-last-phase-transition-time"]
+==== Persistent volume last phase transition time parameter (Technology Preview)
+In {product-title} 4.16 introduces a new parameter, `LastPhaseTransitionTime`, which has a timestamp that is updated every time a persistent volume (PV) transitions to a different phase (`pv.Status.Phase`). This feature is being released with Technology Preview status.
+
+[id="ocp-4-16-storage-csi-smb-cifs-driver-operator"]
+==== Persistent storage using SMB CIFS CSI Driver Operator (Technology Preview)
```

Comment:
```suggestion
==== Persistent storage using CIFS/SMB CSI Driver Operator (Technology Preview)

```
Maybe better to keep the same name with the display name https://github.com/openshift/csi-operator/blob/79ca0ac2df11f8e047e3eae4e08f13bf745814f1/Dockerfile.samba#L10 .

Reply:
+1

Reply:
Fixed.